cli/src/test/java/com/liferay/blade/cli/command/ServerStartCommandTest.java

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Can you collapse both of these two tests into one class called `ServerCommandsTest` and then have two methods in there, testing `server start` and `server stop`?